examples/generate_random_structures/get_initial_structure.py

```diff
@@ -32,37 +32,47 @@
 # Particle definitions
 mass = 100.0 * unit.amu
 masses = {"backbone_bead_masses": mass, "sidechain_bead_masses": mass}
-r_min = 3.0 * bond_length  # Lennard-Jones potential r_min
+r_min = 2.0 * bond_length  # Lennard-Jones potential r_min
 sigma = r_min / (2.0 ** (1 / 6))  # Factor of /(2.0**(1/6)) is applied to convert r_min to sigma
 sigmas = {"bb_sigma": sigma, "sc_sigma": sigma}
-epsilon = 0.5 * unit.kilocalorie_per_mole
+epsilon = 0.1 * unit.kilojoule_per_mole
 epsilons = {"bb_eps": epsilon, "sc_eps": epsilon}
 
 # Bond angle definitions
-bond_angle_force_constant = 0.5 * unit.kilocalorie_per_mole / unit.radian / unit.radian
+bond_angle_force_constant = 15.0 * unit.kilojoules_per_mole / unit.radian / unit.radian
 bond_angle_force_constants = {
     "bb_bb_bb_angle_k": bond_angle_force_constant,
     "bb_bb_sc_angle_k": bond_angle_force_constant,
+    "bb_sc_sc_angle_k": bond_angle_force_constant,
 }
-bb_bb_bb_equil_bond_angle = 120.0 * (
-    3.14 / 180.0
-)  # OpenMM requires angle definitions in units of radians
-bb_bb_sc_equil_bond_angle = 120.0 * (3.14 / 180.0)
+bb_bb_bb_equil_bond_angle = 120.0 * unit.degrees
+bb_bb_sc_equil_bond_angle = 120.0 * unit.degrees
 equil_bond_angles = {
     "bb_bb_bb_angle_0": bb_bb_bb_equil_bond_angle,
     "bb_bb_sc_angle_0": bb_bb_sc_equil_bond_angle,
+    "bb_sc_sc_angle_0": bb_bb_sc_equil_bond_angle,
```

Comment:
Should we set bb_sc_sc_equil_bond_angle = 120.0 * unit.degrees explicitly like the others so it could be changed independently?

Reply:
I don't know that it really matters.  get_initial_structure.py is just an example to have something that works. It does look like a typo, though, so I can tweak that.

Reply:
Ok yeah it's not important. The latest change has bb_sb_sc, should be bb_sc_sc.